run.py

```diff
@@ -272,7 +279,8 @@ def plot_samples(epoch):
             # Save figure to logs dir
             plt.savefig(os.path.join(
                 plots_dir,
-                "memory_sample_{}".format(dt.datetime.now().strftime("%d-%mT%H:%M"))
+                "memory_sample_{}".format(
```

Comment:
Używasz jakiegoś autopep czy coś? Bo długość linii na 80 znaków jest wg mnie trochę przesadnie restrykcyjna, jak tutaj na przykład. Ciężko też przeglądać zmiany jak większość to połamanie linii z innych zmian. Może lepiej na 100 ustawić albo coś takiego?

Reply:
Użyłem zwyczajowo autopep8. Jeżeli 80 znaków to za mało to można zwiększyć limit. Co to przeglądania zmian to to jest narzut jednorazowy. Jak raz przeora się kod przez autopep8 to potem zmiany będą dotyczyły tylko obecnych zmian.

Reply:
Niekoniecznie jednorazowy, bo ja i inni mogą nie używać, więc każdy twój commit będzie naprawiał nasze.

Wg mnie długość linii 100 albo nawet 120 ok.

Reply:
Można dodać plik konfiguracyjny dla pepa i tam określić długość linii. Wtedy taki plik mamy w repo i wszystkim formatuje jednakowo.

Reply:
Nom, ja też używam autopep8 i mam na 100 znaków. Trzeba to koniecznie ujednolicić! Dodasz ten plik konfiguracyjny @Matioz? I wtedy cofnij te łamania linii plz ❤️